std_msgs/Header header

SelfColor self_color
WorkMode work_mode
BulletSpeed bullet_speed

float32 roll
float32 pitch
float32 yaw

float32 control_id

float32 game_progress 
float32 current_hp
float32 current_base_hp 		
float32 allow_fire_amount  	
float32 current_outpost_hp  	
float32 current_enemy_base_hp   
float32 current_enemy_outpost_hp  

float32 hero_x 
float32 hero_y 
float32 cmd_x 
float32 cmd_y 